modules/core/data_stream/modules/notification/data_stream_notification.info.yml

```diff
@@ -5,3 +5,4 @@ package: farmOS
 core_version_requirement: ^9
 dependencies:
   - farm:data_stream
+  - farm:farm_notification
```

Comment:
I think we need an update hook to enable `farm_notification` on sites that already have `data_stream_notification` ?